angularjs/angular.d.ts

```diff
@@ -453,7 +453,7 @@ declare module angular {
          * following compilation. The observer is then invoked whenever the
          * interpolated value changes.
          */
-        $observe(name: string, fn: (value?: any) => any): Function;
+        $observe<T>(name: string, fn: (value?: T) => any): Function;
```

Comment:
Don't see the point of making it generic, Just annotate fn's parameter with a type annotation and you get the *same* level of safety